README.md

```diff
@@ -51,19 +51,19 @@ functionality.) You would want to run Zebra if you want to contribute to the
 Zcash network: the more nodes are run, the more reliable the network will be
 in terms of speed and resistance to denial of service attacks, for example.
 
-These are some of advantages or benefits of Zebra:
+These are some of the advantages or benefits of Zebra:
 
-- Better performance: since it was implemented from scratch, Zebra was able to be
-  implemented in a manner that is currently faster than `zcashd`.
+- Better performance: since it was implemented from scratch in an async, parallelized way, Zebra 
+  is currently faster than `zcashd`.
 - Better security: since it is developed in a memory-safe language (Rust), Zebra
-  is less likely to be affected by security bugs that could compromise the
-  environment where it is run.
+  is less likely to be affected by memory-safety and correctness security bugs that 
+  could compromise the environment where it is run.
 - Better governance: with a new node deployment, there will be more developers
-  who can implement different features.
-- Dev accessibility: there will be more developers which gives new developers
-  options for contributing to protocol development.
-- Runtime safety: the detection of consensus bugs can happen quicker, preventing
-  the likelihood of black swan events.
+  who can implement different features for the Zcash network.
+- Dev accessibility: supports more developers, which gives new developers
+  options for contributing to Zcash protocol development.
+- Runtime safety: with an independeny implementation, the detection of consensus bugs 
+  can happen quicker, preventing the likelihood of black swan events.
```

Comment:
Pre-Existing Issue: this sentence uses an idiom that is difficult to understand - let's state the advantage directly

```suggestion
  can happen quicker, reducing the risk of consensus splits.
```